CHANGELOG.md

```diff
@@ -25,6 +26,10 @@ v0.31.0-rc.0 (2023-01-26)
   - `agentctl` is now `grafana-agentctl`.
   - `agent-operator` is now `grafana-agent-operator`.
 
+- Node Exporter configuration options changed to align with new upstream version (@Thor77):
```

Comment:
This should be in Main (unreleased) section

Reply:
Oh, I messed this up while rebasing, fixed.